tests/unit/badge/badge.renderBadge.spec.js

```diff
@@ -0,0 +1,42 @@
+import { shallow } from 'enzyme';
```

Comment:
I also thought that way, but concentrating all in one file, mostly grouped by rendering differences and/or interactions, seems waaay more reasonable :)